SQLScripts/20211103_UpdateResetPasswordIdForTests.sql

```diff
@@ -0,0 +1 @@
+UPDATE Candidates SET ResetPasswordID = 1 WHERE CandidateID = 2
```

Comment:
I believe the ResetPasswordID column in the Candidates table has a foreign key constraint on it. Can we guarantee that there will be a ResetPassword record with an ID of 1? or should it perhaps be added along with this as well?